src/hub.html

```diff
@@ -90,7 +91,35 @@
                         </template>
 
                         <template data-selector=".Chest">
-                            <a-entity personal-space-invader="radius: 0.2; useMaterial: true;" bone-visibility></a-entity>
+                            <a-entity>
+                              <a-entity personal-space-invader="radius: 0.2; useMaterial: true;" bone-visibility> </a-entity>
+                              <a-entity billboard>
+                                  <a-entity
+                                      block-button
+                                      visible-while-frozen
+                                      text-button="haptic:#player-right-controller;
+                                                   textEl:.text;
```

Comment:
You could use `[text]` selector here instead to avoid the class, not a big deal either way

Reply:
oh yea, I forgot about that